Module-6-Assignment/ClassDeletionTriggerTest.cls

```diff
@@ -0,0 +1,47 @@
+/**
+* Test class to test ClassDeletionTrigger.
+*/
+@isTest
+public class ClassDeletionTriggerTest {
+    
+    @isTest
+    private static void testClassDeletion1()
```

Comment:
change the method name to properly reflect what the test is doing

Reply:
Done